R/Design-class.R

```diff
@@ -409,110 +409,118 @@ DualResponsesSamplesDesign <- function(eff_model,
   )
 }
 
-# nolint start
-
-## ---------------------------------------------------------------------------------------------------
-## class for design based on DLE and efficacy response without  samples using pseudo DLE and efficacy models
-## ----------------------------------------------------------------------------------------------------
-##' This is a class of design based on DLE responses using the \code{\linkS4class{LogisticIndepBeta}} model
-##' model and efficacy responses using \code{\linkS4class{ModelEff}}  model class
-##' without DLE and efficacy samples. It contain all slots in
-##' \code{\linkS4class{RuleDesign}} and \code{\linkS4class{TDDesign}} class object
-##'
-##' @slot data the data set of \code{\linkS4class{DataDual}} class object
-##' @slot Effmodel the pseudo efficacy model to be used, an object class of
-##' \code{\linkS4class{ModelEff}}
-##'
-##' @example examples/design-class-DualResponsesDesign.R
-##' @export
-##' @keywords class
+# DualResponsesDesign ----
+
+## class ----
+
+#' `DualResponsesDesign`
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' This is a class of design based on DLE responses using the [`LogisticIndepBeta`] model
+#' model and efficacy responses using [`ModelEff`]  model class
+#' without DLE and efficacy samples. It contain all slots in
+#' [`RuleDesign`] and [`TDDesign`] class object
+#'
+#' @slot data the data set of [`DataDual`] class object
+#' @slot eff_model the pseudo efficacy model to be used, an object class of
+#' [`ModelEff`]
+#'
+#' @export
+#'
 .DualResponsesDesign <-
   setClass(
     Class = "DualResponsesDesign",
-    representation(
-      Effmodel = "ModelEff",
+    representation = list(
```

Comment:
```suggestion
    slots = c(
```

Reply:
As you suggested, I will keep all DualResponsesDesign as it was in main branch. So this change will not be implemented.